.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -86,18 +83,25 @@ body:
       feature below, *use your best guess if unsure*.
 
 
-      **Tip:** Be advised that the source for some parts of the
-      documentation are hosted outside of this repository. If the page
-      you are reporting describes modules/plugins/etc that are not
-      officially supported by the Ansible Core Engineering team, there
-      is a good chance that it is coming from one of the [Ansible
-      Collections maintained by the community][collections org]. If this
-      is the case, please make sure to file an issue under the
-      appropriate project there instead.
+      **Tip:** Source files for Ansible community documentation are
+      hosted in different repositories. Please make sure to file an
+      issue in the correct project.
+      Documentation for modules/plugins/etc that are officially
+      supported by the Ansible Core Engineering team is available
+      in this (`ansible/ansible`) repository.
+      The Installation Guide, Playbook Guide, Developer Guide,
+      and other documentation is available in the
+      [`ansible/ansible-documentation`] repository.
+      Documentation for other modules/plugins/etc is likely to be
+      available in one of the [Ansible Collections maintained by
+      the community][collections org]. Select the **Issue Tracker** button
```

Comment:
I would link to the Collection Index (https://docs.ansible.com/ansible/latest/collections/index.html) instead of the ansible-collections Github org.